src/api/eth/get_network.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * Returns current network id and name
+ * see:
+ * https://github.com/MetaMask/faq/blob/master/DEVELOPERS.md#construction_worker-network-check
+ *
+ **/
+const P = require('bluebird')
+
+module.exports = async (efx) => {
+  const get = P.promisify(efx.web3.version.getNetwork)
```

Comment:
@prdn / @plutoegg the idea here is to export a util function so we could for instance load a different default configuration based on the eth network we are connected to.